crates/astria-core/src/protocol/transaction/v1alpha1/mod.rs

```diff
@@ -282,6 +327,10 @@ enum UnsignedTransactionErrorKind {
     Action(#[source] action::ActionError),
     #[error("`params` field is unset")]
     UnsetParams(),
+    #[error("invalid type URL")]
+    InvalidTypeUrl,
```

Comment:
Useful to provide a bit more info:
```suggestion
    #[error("encountered invalid type URL when converting from `google.protobuf.Any`; got `{got}`, expected `{}`", UnsignedTransaction::type_url())]
    InvalidTypeUrl { got: String, },
```